src/interface/interface.ts

```diff
@@ -174,14 +174,14 @@ export interface Round1Info {
 
   // target_instances is the original aggregate proofs output batch instances
   // it equivalent to each Round1BatchProof.batch_instances
-  target_instances: number[][];
+  target_instances: Uint8Array[];
```

Comment:
In examples/src/verify/verifyAutoSubmitProofs, no need any "Convert ..." any more?

Reply:
Just quick search they [] and Array<> are completely equivalent, just better to unify the codes